jupyter_book/build.py

```diff
@@ -258,13 +258,16 @@ def build_book(path_book, path_toc_yaml=None, path_ssg_config=None,
         yaml_fm += ["has_widgets: {}".format(has_widgets)]
 
         # Page metadata
-        yaml_fm += ["title: '{}'".format(title)]
+        yaml_fm += ["title: |-"]
```

Comment:
```suggestion
        # Use YAML block scalars for titles so that people can use special characters
        # See http://blogs.perl.org/users/tinita/2018/03/strings-in-yaml---to-quote-or-not-to-quote.html
        yaml_fm += ["title: |-"]
```